packages/config/config/devices/0x015f/mh-s212.json

```diff
@@ -21,5 +25,188 @@
 			"maxNodes": 5,
 			"isLifeline": true
 		}
+	},
+	"supportsZWavePlus": true,
```

Comment:
We don't use `supportsZWavePlus` anymore

Reply:
OK have removed